### PR TITLE
New: EuSoft.Eudic version 12.6.5

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25448)

### CODE REVIEW

File: manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: EuSoft.Eudic
+PackageVersion: 12.6.5
```
Comment (OfficialEsco) [inline manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.installer.yaml:5]:
```suggestion
PackageVersion: 12.0.0.0
```

File: manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: EuSoft.Eudic
+PackageVersion: 12.6.5
```
Comment (OfficialEsco) [inline manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 12.0.0.0
```

File: manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EuSoft.Eudic
+PackageVersion: 12.6.5
```
Comment (OfficialEsco) [inline manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 12.0.0.0
```

File: manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+PackageLocale: en-US
+Publisher: EuSoft
+#PublisherUrl: 
+PublisherSupportUrl: https://www.eudic.net/v4/en/home/support
+PrivacyUrl: https://www.eudic.net/v4/home/privacy
+#Author: 
+PackageName: Eudic
```
Comment (OfficialEsco) [inline manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml:12]:
```suggestion
Publisher: 欧路软件
#PublisherUrl: 
PublisherSupportUrl: https://www.eudic.net/v4/en/home/support
PrivacyUrl: https://www.eudic.net/v4/home/privacy
#Author: 
PackageName: 欧路词典
```

File: manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+PackageUrl: https://www.eudic.net/v4/en/app/eudic
+License: Freeware
+#LicenseUrl: 
+Copyright: Copyright (c) 2021 EuSoft
+#CopyrightUrl: 
+ShortDescription: An authoritative English dictionary software and essential tool for English learners
+Description: An authoritative English dictionary software and essential tool for English learners, providing you with real-life pronunciation of words, English translation, cross-software word retrieval, crossword search, English expanded thesaurus, English word memorization, English listening, real-time English radio, etc.
+Tags:
```
Comment (OfficialEsco) [inline manifests/e/EuSoft/Eudic/12.6.5/EuSoft.Eudic.locale.en-US.yaml:20]:
```suggestion
Moniker: Eudic
Tags:
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hey there, the Software is reporting the incorrect version in Windows `A`dd or `R`emove `P`rograms which winget requires to match..
image

Also since the Software PackageName is still `欧路软件` on a en-US OS we will have to use that too.

**Comment by SpecterShell** [comment]
Oh, I forget about it.

**Comment by OfficialEsco** [comment]
```
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\e\EuSoft\Eudic\12.6.5', 
expected 'manifests\e\EuSoft\Eudic\12.0.0.0'
```

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**